addons/etcd/scripts/switchover-without-candidate.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+set -ex
+
+leaderEndpoint=${KB_LEADER_POD_FQDN}:2379
+candidateEndpoint=""
+
+# see common.sh, this function may change leaderEndpoint
+updateLeaderIfNeeded 3
+
```

Comment:
if, after updateLeaderIfNeeded, leaderEndpoint doesn't equal the one specified in KB_LEADER_POD_FQDN, you may consider this "switchover-without-candidate" operation complete, and it can return immediately.

Reply:
done